spec/ruby-lint/analysis/undefined_variables_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'spec_helper'
+require_relative '../../spec_helper'
```

Comment:
This should not be changed. RSpec takes care of adding `spec` to the load path so `require_relative` is not needed.